coin_prices_from_coingecko/go.mod

```diff
@@ -1,3 +1,9 @@
 module github.com/blocky/as-demo
 
-go 1.22
+go 1.22.6
+
+require (
+	github.com/blocky/basm-go-sdk v0.0.0-20250312001256-99b377e30606 // indirect
```

Comment:
something seems out of date with this `go.mod` file.  Since `main.go` imports the go sdk, it should be a direct dependency.  I think running `go mod tidy` should fix.